File: spring-data-requery-java/src/test/java/com/coupang/springframework/data/requery/domain/sample/AbstractConcreteType2.java
package com.coupang.springframework.data.requery.domain.sample;

import io.requery.Entity;

/**
 * AbstractConcreteType2
 *
 * @author debop@coupang.com
 * @since 18. 6. 25
 */
@Entity
public abstract class AbstractConcreteType2 extends AbstractMappedType {

    public AbstractConcreteType2() {}

    public AbstractConcreteType2(String attribute1) {
        super(attribute1);
    }

    private static final long serialVersionUID = 6778710401940951022L;


}

File: spring-data-requery-kotlin/src/test/kotlin/com/coupang/springframework/data/requery/domain/sample/AbstractConcreteType2.kt
package com.coupang.springframework.data.requery.domain.sample

import io.requery.Entity
import io.requery.Generated
import io.requery.Key

@Entity
abstract class AbstractConcreteType2(): AbstractMappedType() {

    constructor(attribute1: String?): this() {
        this.attribute1 = attribute1
    }

    @get:Key
    @get:Generated
    abstract val id: Long


}
